bundles/org.openhab.binding.enocean/src/main/resources/OH-INF/i18n/enocean.properties

```diff
@@ -184,18 +184,18 @@ thing-type.config.enocean.measurementSwitch.receivingEEPId.option.D2_01_00 = Swi
 thing-type.config.enocean.measurementSwitch.receivingEEPId.option.D2_01_01 = Switching (EEP: D2_01_01)
 thing-type.config.enocean.measurementSwitch.receivingEEPId.option.D2_01_02 = Switching, dimming and energy measurement (EEP: D2_01_02)
 thing-type.config.enocean.measurementSwitch.receivingEEPId.option.D2_01_03 = Switching and dimming (EEP: D2_01_03)
-thing-type.config.enocean.measurementSwitch.receivingEEPId.option.D2_01_04 = Switching, dimming and energy measurement (EEP: D2_01_04)
-thing-type.config.enocean.measurementSwitch.receivingEEPId.option.D2_01_05 = Switching, dimming and energy measurement (EEP: D2_01_05)
+thing-type.config.enocean.measurementSwitch.receivingEEPId.option.D2_01_04 = Switching, configurable dimming with power and energy measurement (EEP: D2_01_04)
+thing-type.config.enocean.measurementSwitch.receivingEEPId.option.D2_01_05 = Switching, configurable dimming with power and energy measurement (EEP: D2_01_05)
 thing-type.config.enocean.measurementSwitch.receivingEEPId.option.D2_01_06 = Switching and energy measurement (EEP: D2_01_06)
 thing-type.config.enocean.measurementSwitch.receivingEEPId.option.D2_01_07 = Switching (EEP: D2_01_07)
-thing-type.config.enocean.measurementSwitch.receivingEEPId.option.D2_01_08 = Switching, dimming and energy measurement (EEP: D2_01_08)
-thing-type.config.enocean.measurementSwitch.receivingEEPId.option.D2_01_09 = Switching, dimming and energy measurement (EEP: D2_01_09)
+thing-type.config.enocean.measurementSwitch.receivingEEPId.option.D2_01_08 = Switching with power and energy measurement (EEP: D2_01_08)
+thing-type.config.enocean.measurementSwitch.receivingEEPId.option.D2_01_09 = Switching, configurable dimming with power and energy measurement (EEP: D2_01_09)
 thing-type.config.enocean.measurementSwitch.receivingEEPId.option.D2_01_09_PERMUNDO = Permundo PSC
-thing-type.config.enocean.measurementSwitch.receivingEEPId.option.D2_01_0A = Switching and energy measurement (EEP: D2_01_0A)
-thing-type.config.enocean.measurementSwitch.receivingEEPId.option.D2_01_0B = Switching and energy measurement (EEP: D2_01_0B)
-thing-type.config.enocean.measurementSwitch.receivingEEPId.option.D2_01_0C = Switching and energy measurement (EEP: D2_01_0C)
+thing-type.config.enocean.measurementSwitch.receivingEEPId.option.D2_01_0A = Switching and power failure detection (EEP: D2_01_0A)
+thing-type.config.enocean.measurementSwitch.receivingEEPId.option.D2_01_0B = Switching, power failure detection with power and energy measurement (EEP: D2_01_0B)
+thing-type.config.enocean.measurementSwitch.receivingEEPId.option.D2_01_0C = Switching, pilot wire with power and energy measurement (EEP: D2_01_0C)
```

Comment:
```suggestion
thing-type.config.enocean.measurementSwitch.receivingEEPId.option.D2_01_0C = Switching, pilot wire mode, power failure detection, power and energy measurement (EEP: D2_01_0C)
```

Reply:
You are right `D2_01_0C` doesn't have Power failure detection (the table is quite difficult to read - I mixed the columns).